x/metadata/client/cli/tx.go

```diff
@@ -309,24 +313,24 @@ func WriteContractSpecificationCmd() *cobra.Command {
 		Use:   "write-contract-specification [contractspec-id] [owners] [parties-involved] [source-value] [classname] [description-name] [description] [website-url] [icon-url]",
 		Short: "Add/Update metadata contract specification on the provenance blockchain",
 		Long: `Add/Update metadata contract specification on the provenance blockchain
-[contractspec-id] - contract specification metaaddress
-[owners] - comma delimited list of bech32 owner addresses
-[parties-involved] - comma delimited list of party types.  Accepted values: originator,servicer,investor,custodian,owner,affiliate,omnibus,provenance
-[source-value] - source identifier of type hash or resourceid
-[classname] - Name of contract specification
-[description-name] - optional- description name identifier
-[description] - optional - description text
-[website-url] - optional - address of website
-[icon-url] - optional - address to a image to be used as an icon
-		`,
+contractspec-id]   - contract specification metaaddress
```

Comment:
Looks like a '[' got chopped off the beginning.
```suggestion
[contractspec-id]   - contract specification metaaddress
```

Reply:
Thanks, fixed it.